test/controllers/actions/AuthActionSpec.scala

```diff
@@ -213,7 +213,6 @@ object AuthActionSpec {
 
   private def appConfig(isSchemeOverviewEnabled: Boolean = false) = {
     val application = new GuiceApplicationBuilder()
```

Comment:
I removed this configure statement as it is not needed anymore. Please pull latest on your code.